app/DTO/User.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\DTO;
+
+use App\Models\User as UserModel;
+
+/**
+ * Data Transfer Object (DTO) to transmit the attributes of a User.
+ */
+class User extends ArrayableDTO
```

Comment:
Why do we need this extra DTO? I get the point for albums and photos, because those DTOs decorate the actual album or photo with the currently effective rights of the authenticated user upon those objects. But nothing of this sort happens here. If I don't miss something, this DTO is simply a 1:1 copy of the user model.